doc/devel/development_workflow.rst

```diff
@@ -231,12 +173,9 @@ To see a linear list of commits for this branch::
 
    git log
 
-You can also look at the `network graph visualizer`_ for your github
+You can also look at the `network graph visualizer <https://github.com/matplotlib/matplotlib/network>`_ for your github
```

Comment:
We should remove this, the view for Matplotlib is approximately useless (we have in the past asked GitHub ahout it and their response was "ahhh, your network is too big....")